doc/logging.md

```diff
@@ -0,0 +1,17 @@
+# How does Happy test log work
+
+## Types of Happy logs
+  * Happy debug logs — Happy will run as test coordination and orchestrator, facilitating the activities of the virtual test device nodes. Happy debug logs will have logs from Happy tool only.
+
+  * Happy device logs — Happy will host virtual network which has multiple virtual devices, Happy device logs will have logs for software running on the virtual device, OpenWeave is one of the software example, so device logs will have OpenWeave logs.
+
+## Where Happy logs are stored
+
+Both Happy host and device logs location are processed in `happy/Driver.py` as below:
+  * If `HAPPY_LOG_DIR` is found in os environment variable, Happy uses `HAPPY_LOG_DIR` value as log folder location to store Happy test logs. For example, OpenWeave tests store Happy test logs along with feature folder under build folder.
+  * If `HAPPY_LOG_DIR` is not found in os environment variable, Happy uses `default_happy_log_dir` value from [happy/conf/main_config.json](https://github.com/openweave/happy/blob/master/happy/conf/main_config.json) as the default log folder location to store Happy test logs. The default value of `default_happy_log_dir` is `/tmp`, user can modify it if they want.
```

Comment:
Thank you for clarifying the default value of default_happy_log_dir in main_config.json. However, if someone is using an installed Happy distribution, main_config.json isn't something they can easily override, correct? This is really only something someone could / should modify if they were working on a "source" distribution of Happy, correct?

If so, this again seems like an implementation detail that shouldn't be included in this section of the documentation.